numpy/core/src/private/lowlevel_strided_loops.h

```diff
@@ -414,26 +414,35 @@ PyArray_PrepareThreeRawArrayIter(int ndim, npy_intp *shape,
                             char **out_dataC, npy_intp *out_stridesC);
 
 /*
- * Return number of elements that must be peeled from
- * the start of 'addr' with 'nvals' elements of size 'esize'
- * in order to reach 'alignment'.
- * alignment must be a power of two.
- * see npy_blocked_end for an example
+ * Return number of elements that must be peeled from the start of 'addr' with
+ * 'nvals' elements of size 'esize' in order to reach 'alignment'.  alignment
+ * must be a power of two.  see npy_blocked_end for an example
+ *
+ * 16-12-2017 Changed to use npy_intp for return type and nvals. That
```

Comment:
This is a very weird date format. I'd also question the practice of adding comments like this in the presence of version control with good commit messages.

Reply:
Can take it out, it is the sort of thing folks used to leave in code, but I feel no particular attachment to it.

Reply:
I think it might still be worth including the rationale for not using unsigned values in the comment - just not the history.